.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.2
```

Comment:
These should be `3.3.6`. This worked fine for me when I did that.

Reply:
Huh, how'd that not get committed, I definitely changed it but must have discarded the change by mistake. Weird. I'll give that a shot.

Reply:
Updated.  This now works for me.